### PR TITLE
2021 03 23 deeamon128 navigation menu

### PR DESCRIPTION
**Pull request by deeamon128**

### CODE REVIEW

**Review by biggicode — APPROVED** [review]

**Review by marianzburlea — APPROVED** [review]
💯

File: web/deeamon128/html/navigation-menu/app.js [hunk 1]
```diff
@@ -0,0 +1,7 @@
+const menu = document.querySelector('#mobile-menu');
+const menuLinks = document.querySelector ('.nav-menu');
```
Comment (marianzburlea) [inline web/deeamon128/html/navigation-menu/app.js:2]: remove space before (

`const menuLinks = document.querySelector('.nav-menu');`

File: web/deeamon128/html/navigation-menu/style.css [hunk 2]
```diff
@@ -0,0 +1,185 @@
+:root {
+  --pickled-bluewood: #2B4448;
+  --rolling-stone: #758086;
+  --burnt-sienna: #FFB9B9;
+  --white: #FFFFFF;
+}
```
Comment (marianzburlea) [inline web/deeamon128/html/navigation-menu/style.css:6]: Nice :)

File: web/deeamon128/html/navigation-menu/style.css [hunk 3]
```diff
@@ -0,0 +1,185 @@
...
+
+.nav-container {
+  width: 100%;
+  background-color: var(--pickled-bluewood);
+}
+
+.navbar {
+  display: grid;
+  grid-template-columns: 0.2fr auto 1fr;
```
Comment (marianzburlea) [inline web/deeamon128/html/navigation-menu/style.css]: you can change
`  grid-template-columns: 0.2fr auto 1fr;`

to 

`  grid-template-columns: 1fr auto 5fr;`

File: web/deeamon128/html/navigation-menu/style.css [hunk 4]
```diff
@@ -0,0 +1,185 @@
...
+}
+
+.nav-links {
+  color: var(--white);
+  text-decoration: none;
+}
+
+.nav-links:hover {
+  color: var(--burnt-sienna);
+  transition: all 0.2s, ease-out;
```
Comment (marianzburlea) [inline web/deeamon128/html/navigation-menu/style.css]: remove the comma `  transition: all 0.2s ease-out;`

File: web/deeamon128/html/navigation-menu/style.css [hunk 5]
```diff
@@ -0,0 +1,185 @@
...
+  padding: 6px 16px;
+  border-radius: 4px;
+}
+
+.nav-links-btn:hover {
+  background-color: transparent;
+  border: 1px solid var(--burnt-sienna);
+  padding: 5px 15px;
+  border-radius: 4px;
+  transition: all 0.2s, ease-out;
```
Comment (marianzburlea) [inline web/deeamon128/html/navigation-menu/style.css]: remove the comma

File: web/deeamon128/html/navigation-menu/style.css [hunk 6]
```diff
@@ -0,0 +1,185 @@
...
+}
+
+.menu-toggle .bar {
+  width: 25px;
+  height: 3px;
+  margin: 5px auto;
+  transition: all 0.3s ease-in-out;
```
Comment (marianzburlea) [inline web/deeamon128/html/navigation-menu/style.css:73]: nice :), no comma

**Review by marianzburlea — APPROVED** [review]
💯

File: web/deeamon128/html/navigation-menu/app.js (hunk 1 above)
Comment (deeamon128) [reply web/deeamon128/html/navigation-menu/app.js:2]: thanks for your feedback :D

**Review by BogdanNegura — APPROVED** [review]
👍

**Review by robertoalx — APPROVED** [review]
💯

**Review by RadosavAntonio — APPROVED** [review]
👌🏻

**Pull request merged**